src/brogue/MainMenu.c

```diff
@@ -237,136 +247,373 @@ void initializeMenuFlames(boolean includeTitle,
+static void chooseGameVariant() {
+    short gameVariantChoice;
+    char textBuf[TEXT_MAX_LENGTH] = "", tmpBuf[TEXT_MAX_LENGTH] = "", goldColorEscape[5] = "", whiteColorEscape[5] = "";
+
+    encodeMessageColor(goldColorEscape, 0, &yellow);
+    encodeMessageColor(whiteColorEscape, 0, &white);
+
+    snprintf(textBuf, TEXT_MAX_LENGTH, "%sBrogue%s\n", goldColorEscape, whiteColorEscape);
+    append(textBuf, "Classic Brogue. The endlessly captivating masterpiece of dungeon adventuring.\n\n", TEXT_MAX_LENGTH);
+
+    snprintf(tmpBuf, TEXT_MAX_LENGTH, "%sRapid Brogue%s\n", goldColorEscape, whiteColorEscape);
+    append(textBuf, tmpBuf, TEXT_MAX_LENGTH);
+    append(textBuf, "Die faster and more often in this quarter-length version of the classic game!\n\n", TEXT_MAX_LENGTH);
+
+    brogueButton buttons[2];
+    cellDisplayBuffer rbuf[COLS][ROWS];
+    copyDisplayBuffer(rbuf, displayBuffer);
+    initializeMainMenuButton(&(buttons[0]), "  %sR%sapid Brogue     ", 'r', 'R', NG_NOTHING);
+    initializeMainMenuButton(&(buttons[1]), "     %sB%srogue        ", 'b', 'B', NG_NOTHING);
+    gameVariantChoice = printTextBox(textBuf, 20, 7, 45, &white, &black, rbuf, buttons, 2);
+    overlayDisplayBuffer(rbuf, NULL);
+
+    if (gameVariantChoice == 1) {
+        gameVariant = VARIANT_BROGUE;
+    } else if (gameVariantChoice == 0) {
+        gameVariant = VARIANT_RAPID_BROGUE;
+    } else {
+        rogue.nextGame = NG_NOTHING;
+    }
+}
+
+/// @brief Displays a dialog window for the user to chose a game mode. The game mode is displayed in the bottom left
+/// on the title screen (except normal mode).
+static void chooseGameMode() {
+    short gameMode;
+    char textBuf[TEXT_MAX_LENGTH] = "", tmpBuf[TEXT_MAX_LENGTH] = "", goldColorEscape[5] = "", whiteColorEscape[5] = "";
+
+    encodeMessageColor(goldColorEscape, 0, &yellow);
+    encodeMessageColor(whiteColorEscape, 0, &white);
+
+    snprintf(textBuf, TEXT_MAX_LENGTH, "%sNormal Mode%s\n", goldColorEscape, whiteColorEscape);
+    append(textBuf, "Punishingly difficult. Maliciously alluring. Perfectly normal.\n\n", TEXT_MAX_LENGTH);
+
+    snprintf(tmpBuf, TEXT_MAX_LENGTH, "%sEasy Mode%s\n", goldColorEscape, whiteColorEscape);
+    append(textBuf, tmpBuf, TEXT_MAX_LENGTH);
+    append(textBuf, "Succumb to demonic temptation and play as an all-powerful ampersand, taking 20%% as much damage. "
+                    "But great power comes at a great price -- specifically, a 90% income tax rate.\n\n", TEXT_MAX_LENGTH);
+
+    snprintf(tmpBuf, TEXT_MAX_LENGTH, "%sWizard Mode%s\n", goldColorEscape, whiteColorEscape);
+    append(textBuf, tmpBuf, TEXT_MAX_LENGTH);
+    append(textBuf, "Play as an invincible wizard that starts with legendary items and is magically reborn after every "
+                    "death. Summon monsters and make them friend or foe. Conjure any item out of thin air. All this and"
+                    " more, for the bargain basement price of forfeiting your score.", TEXT_MAX_LENGTH);
```

Comment:
unsure on this replacement, bit boring, but just wanted to steer away from idioms/references

```suggestion
    append(textBuf, "Play as an invincible wizard that starts with legendary items and is magically reborn after every "
                    "death. Summon monsters and make them friend or foe. Conjure any item out of thin air. "
                    "(Your score is not saved.)", TEXT_MAX_LENGTH);
```